contrib/nodejs/buffer.js

```diff
@@ -55,3 +55,11 @@ buffer.SlowBuffer.prototype.slice;
  * @return {string}
  */
 buffer.SlowBuffer.prototype.toString;
+
+/**
+ * @param {number} size
+ * @param {(string|Buffer|number)=} fill
```

Comment:
Write `!Buffer` to make it non-nullable.

Reply:
Will do.